### ISSUE TITLE
Heston model implied vol

### ISSUE THREAD

**Issue by dancixx** [nakashima-hikaru/implied-vol#5]
Hi there,

I tried to find a paper on how to implement implied volatility under the Heston model. Do you have any suggestions or know of a good research paper? I have only found asymptotic proofs, but I'm not sure if that's what I need.

And one more package related question, that the method `implied_black_volatility` this calculates the IV under the basic BSM model, right?

**Comment by faysou** [comment]
Yes under BSM. Heston is irrelevant for this package. I'm not the maintainer of this project, but I use it.
